package-res/resources/web/pentaho/visual/base/View.js

```diff
@@ -180,22 +297,107 @@ define([
     //endregion
 
     //region Update
+    /**
+     * Updates the view to match its model's latest state.
+     *
+     * When [isAutoUpdate]{@link pentaho.visual.base.View#isAutoUpdate} is `true`,
+     * any change to the view's model automatically triggers a view update, by calling this method.
+     *
+     * The update operation is _generally_ asynchronous.
+     * Even if the view implementation completes its update synchronously,
+     * the completion is only advertised asynchronously,
+     * through the emission of one of the events
+     * [did:update]{@link pentaho.visual.base.events.DidUpdate} or
+     * [rejected:update]{@link pentaho.visual.base.events.RejectedUpdate} and
+     * the resolution of the returned promise.
+     *
+     * The [isUpdating]{@link pentaho.visual.base.View#isUpdating} property is `true` while the
+     * update operation is considered in progress,
+     * a period during which calling [update]{@link pentaho.visual.base.View#update} again
+     * returns the same promise that was returned by the initial `update` call.
+     *
+     * If the view is not currently updating and is also not [dirty]{@link pentaho.visual.base.View#isDirty},
+     * calling `update` _immediately_ returns a fulfilled promise.
+     *
+     * Otherwise, by this time, [isUpdating]{@link pentaho.visual.base.View#isUpdating} will be `true`,
+     * and the update operation proceeds to the **Will phase**.
+     *
+     * #### Will phase
+     *
+     * Initially, the [_updateWill]{@link pentaho.visual.base.View#_updateWill} method is called,
+     * which by default emits the [will:update]{@link pentaho.visual.base.events.WillUpdate} event.
+     * Either the implementation or the event listeners are allowed to cancel the update or further modify the model.
+     *
+     * If the update is _canceled_, the update is rejected with the cancel reason,
+     * and enters the **Rejected phase**.
      *
-     * Executes [_update]{@link pentaho.visual.base.View#_update} asynchronously in
-     * the will/did/rejected event loop associated with an update of a visualization,
-     * and also creates the visualization DOM node the first time it successfully updates.
+     * #### Loop phase
      *
-     * In order to get the visualization DOM node,
-     * listen to the {@link pentaho.visual.base.events.DidCreate|"did:create"} event.
+     * If the update is not canceled, the view enters an **update loop**,
+     * that only ends when either the view is up to date with the model,
+     * in which case surely [isDirty]{@link pentaho.visual.base.View#isDirty} will be `false`,
```

Comment:
(in which case surely [isDirty]{@link pentaho.visual.base.View#isDirty} will be `false`),